### PR TITLE
{bio}[GCCcore/7.3.0] PEAR v0.9.11

### PR DESCRIPTION
**Pull request by SethosII**
(created using `eb --new-pr`)

### CODE REVIEW

**Comment by vanzod** [comment]
@SethosII Does it have any required dependency in the `foss` toolchain or could it be built with `GCC` instead?

**Comment by boegel** [comment]
Not merging for now due to @vanzod's question...

**Comment by SethosII** [comment]
@boegel @vanzod I changed the toolchain to `GCCcore`.

**Comment by vanzod** [comment]
Test report by @vanzod
**FAILED**
Build succeeded for 0 out of 1 (1 easyconfigs in this PR)
cermis - Linux debian 9.7, Intel(R) Xeon(R) CPU           X5670  @ 2.93GHz, Python 2.7.13
See https://gist.github.com/ba8627a6abb5700aa372ef050ab5a582 for a full test report.

File: easybuild/easyconfigs/p/PEAR/PEAR-0.9.11-GCCcore-7.3.0.eb [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+
+homepage = 'https://cme.h-its.org/exelixis/web/software/%(namelower)s/'
+description = """PEAR is an ultrafast, memory-efficient and highly accurate pair-end read merger. 
+ It is fully parallelized and can run with as low as just a few kilobytes of memory."""
+
+toolchain = {'name': 'GCCcore', 'version': '7.3.0'}
+
+# Needs manual download via web-form,
+# https://www.h-its.org/en/research/sco/software/#NextGenerationSequencingSequenceAnalysis
+# source_urls = ['https://cme.h-its.org/exelixis/web/software/pear/files/']
```
Comment (vanzod) [inline easybuild/easyconfigs/p/PEAR/PEAR-0.9.11-GCCcore-7.3.0.eb]: Since now it can only be manually downloaded, I see little point leaving this URL here

**Comment by vanzod** [comment]
Test report by @vanzod
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
cermis - Linux debian 9.7, Intel(R) Xeon(R) CPU           X5670  @ 2.93GHz, Python 2.7.13
See https://gist.github.com/c7d11ca7c3fc31f3b7f15d524157826b for a full test report.

File: easybuild/easyconfigs/p/PEAR/PEAR-0.9.11-GCCcore-7.3.0.eb (hunk 1 above)
Comment (SethosII) [reply easybuild/easyconfigs/p/PEAR/PEAR-0.9.11-GCCcore-7.3.0.eb]: @vanzod I deleted the line.

**Comment by vanzod** [comment]
Going in, thanks @SethosII!

**Pull request merged**